handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslEngine.java

```diff
@@ -0,0 +1,1951 @@
+            new SSLException("renegotiation unsupported"), ReferenceCountedOpenSslEngine.class, "beginHandshake()");
+    private static final SSLException ENCRYPTED_PACKET_OVERSIZED = ThrowableUtil.unknownStackTrace(
+            new SSLException("encrypted packet oversized"), ReferenceCountedOpenSslEngine.class, "unwrap(...)");
+    private static final Class<?> SNI_HOSTNAME_CLASS;
+    private static final Method GET_SERVER_NAMES_METHOD;
+    private static final Method SET_SERVER_NAMES_METHOD;
+    private static final Method GET_ASCII_NAME_METHOD;
+    private static final Method GET_USE_CIPHER_SUITES_ORDER_METHOD;
+    private static final Method SET_USE_CIPHER_SUITES_ORDER_METHOD;
+    private static final ResourceLeakDetector<ReferenceCountedOpenSslEngine> leakDetector =
+            ResourceLeakDetectorFactory.instance().newResourceLeakDetector(ReferenceCountedOpenSslEngine.class);
+
+    static {
+        AtomicIntegerFieldUpdater<ReferenceCountedOpenSslEngine> destroyedUpdater =
+                PlatformDependent.newAtomicIntegerFieldUpdater(ReferenceCountedOpenSslEngine.class, "destroyed");
+        if (destroyedUpdater == null) {
+            destroyedUpdater = AtomicIntegerFieldUpdater.newUpdater(ReferenceCountedOpenSslEngine.class, "destroyed");
+        }
+        DESTROYED_UPDATER = destroyedUpdater;
+
+        Method getUseCipherSuitesOrderMethod = null;
+        Method setUseCipherSuitesOrderMethod = null;
+        Class<?> sniHostNameClass = null;
+        Method getAsciiNameMethod = null;
+        Method getServerNamesMethod = null;
+        Method setServerNamesMethod = null;
+        if (PlatformDependent.javaVersion() >= 8) {
+            try {
+                getUseCipherSuitesOrderMethod = SSLParameters.class.getDeclaredMethod("getUseCipherSuitesOrder");
+                SSLParameters parameters = new SSLParameters();
+                @SuppressWarnings("unused")
+                Boolean order = (Boolean) getUseCipherSuitesOrderMethod.invoke(parameters);
+                setUseCipherSuitesOrderMethod = SSLParameters.class.getDeclaredMethod("setUseCipherSuitesOrder",
+                        boolean.class);
+                setUseCipherSuitesOrderMethod.invoke(parameters, true);
+            } catch (Throwable ignore) {
+                getUseCipherSuitesOrderMethod = null;
+                setUseCipherSuitesOrderMethod = null;
+            }
+            try {
+                sniHostNameClass = Class.forName("javax.net.ssl.SNIHostName", false,
+                        PlatformDependent.getClassLoader(ReferenceCountedOpenSslEngine.class));
+                Object sniHostName = sniHostNameClass.getConstructor(String.class).newInstance("netty.io");
+                getAsciiNameMethod = sniHostNameClass.getDeclaredMethod("getAsciiName");
+                @SuppressWarnings("unused")
+                String name = (String) getAsciiNameMethod.invoke(sniHostName);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "name" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1481)